server/routes/customer.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import sql from 'mssql';
+import dbConfig from '../dbconfig.js';
+
+const router = express.Router();
+
+router.get('/', function (req, res) {
+  const authenticatedUserId = req.session.authenticatedUser;
+  const getData = async () => {
+    try {
+      let pool = await sql.connect(dbConfig);
+
+      let orderResults = await pool
+        .request()
+        .input('customerId', sql.Int, authenticatedUserId)
+        .query(
+          'SELECT customerId, firstName, lastName, email, phonenum, address, city, state, postalCode, country, userid FROM customer WHERE customerId = @customerId'
+        );
+
+      res.write(JSON.stringify(orderResults));
+      res.end();
+    } catch {
+      (error) => {
+        console.log(error);
+        res.write(error);
+        console.dir(err);
+
+        res.end();
+      };
+    }
+  };
```

Comment:
It's a little confusing defining an async function directly within the `get()` and then immediately calling it. I'd recommend moving this to a function definition outside of the get so that the code is more readable.